src/main/java/io/github/wistefan/mapping/annotations/MappingEnabled.java

```diff
@@ -16,4 +16,9 @@
 	 * Supported entity types for the mapping. E.g. which ngsi-ld entity types can be mapped to the object.
 	 */
 	String[] entityType() default {""};
+
+	/**
+	 * Supported subscription type for the mapping. E.g. which ngsi-ld subscription type can be mapped to the object.
+	 */
```

Comment:
Why cant the entity be used here? Overall, a subscription is also an entity. And as far as I know, there is only one notification type in ngsi-ld. What other types could it be?

Reply:
I thought that subscription is a separate type, and not an entity (because it has its own endpoint at context broker). Ok, then I'll make it as a entity type and update the code accordingly